WordPress/src/main/res/values/strings.xml

```diff
@@ -1899,5 +1900,10 @@
     <string name="login_epilogue_mysites_other">My sites</string>
     <string name="login_google_button_prefix">Or you can</string>
     <string name="login_google_button_suffix">Log in with Google.</string>
+    <string name="login_error_button">Close</string>
+    <string name="login_error_email_not_found">The Google account \'%s\' doesn\'t match any existing account on WordPress.com.</string>
+    <string name="login_error_generic">There was some trouble connecting with the Google account.</string>
+    <string name="login_error_suffix_internal">\nMaybe try again?</string>
```

Comment:
Not sure a question like that is very clear for the user. Better be more explicit.

But, if we're actually not sure ourselves whether the user should try again later or not, let's omit suggesting it.

Reply:
@folletto I understand that that copy is taken from the design iteration. Would you mind if we rephrase them to non question form?

Reply:
It's a bit abstract to me to reconnect that to the screen where it appears, so can I get a screenshot to evaluate it? Context is relevant to see if it works.

Reply:
This is one of the cases that are not easy to replicate so, image this popup https://user-images.githubusercontent.com/3827611/29892211-07a06352-8d8b-11e7-8632-ab5f9b19f9f0.png
but having the text:
> There was some trouble connecting
> with the Google account.
> Maybe try again?
> 
> &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;[CLOSE]

Reply:
That's an excellent alternative to a screenshot! 🌟 

I'm ok in keeping the question there, but if you feel it's off we can either ping Editorial or remove that line. Both are ok for me in this scenario (reason: there's no clear reason forward and it's a modal with limited ability to add more context).

Reply:
Cool, let's remove it. @theck13 would that work OK with you too?

Reply:
To be clear, should we remove `login_error_suffix_internal`, `login_error_suffix_invalid`, or both?

***login_error_suffix_internal***
An internal Google API error occurred.  We don't know what happened.
> There was some trouble connecting
> with the Google account.
> Maybe try again?

***login_error_suffix_invalid***
The user attempted to connect with an invalid account name specified.  I'm not sure how a user would do this, but it's recommended by Google to check for it.
> There was some trouble connecting
> with the Google account.
> Maybe try a different account?

Reply:
The second seems a good positive hint for the user to try. These two work well as they are "symmetric", but the "Maybe try again?" is generic, and probably it's a kind of symmetry nobody would ever notice... 

tl;dr: remove "Maybe try again?" — keep "Maybe try a different account?"

Reply:
I think that's best.  I removed `login_error_suffix_internal`  in d042ff0f0d3dc2639cd1377de6b327b247816eb6.